flytekit/clients/friendly.py

```diff
@@ -1046,3 +1048,35 @@ def get_data(self, flyte_uri: str) -> _data_proxy_pb2.GetDataResponse:
 
         resp = self._dataproxy_stub.GetData(req, metadata=self._metadata)
         return resp
+
+    def get_download_deck_signed_url(
+        self,
+        node_id: str,
+        project: str,
+        domain: str,
+        name: str,
+        org: str = "",
+        expires_in: datetime.timedelta = None,
+    ) -> _data_proxy_pb2.CreateDownloadLinkResponse:
+        """
+        This is a new API for flyte and union cluster to get the signed url for the deck artifact.
+        """
+        expires_in_pb = None
+        if expires_in:
+            expires_in_pb = Duration()
+            expires_in_pb.FromTimedelta(expires_in)
+        return super(SynchronousFlyteClient, self).create_download_link(
+            _data_proxy_pb2.CreateDownloadLinkRequest(
+                artifact_type=ARTIFACT_TYPE_DECK,
+                node_execution_id=_identifier_pb2.NodeExecutionIdentifier(
+                    node_id=node_id,
+                    execution_id=_identifier_pb2.WorkflowExecutionIdentifier(
+                        project=project,
+                        domain=domain,
+                        name=name,
+                        org=org,
```

Comment:
ditto

Reply:
I'll remove the argument `org` if this is not required.